cupyx/scipy/fft/fft.py

```diff
@@ -25,15 +26,15 @@ def __getattr__(self, name):
             return None
 
     _scipy_fft = _DummyModule()
-
-try:
-    from _scipy_fft._lib._pep440 import Version
-except ImportError:
-    # either _scipy_fft is _DummyModule, or scipy < 1.4.0
-    _scipy_150 = False
 else:
-    _scipy_150 = Version(_scipy_fft.version.version) >= Version('1.5.0')
-    del Version
+    try:
+        from numpy.lib import NumpyVersion as Version
```

Comment:
nit: The line could be outside of the `try` block.  How about the following to handle both `_scipy_fft` and `_scipy_150`?
```python
try:
    import scipy
    import scipy.fft as _scipy_fft
except ImportError:
    ...
else:
    from numpy.lib import NumpyVersion as Version
    ...
```

Reply:
done. please take a look again